src/mapper/pkg/clouduploader/cloud_upload.go

```diff
@@ -70,7 +48,7 @@ func (c *CloudUploader) uploadDiscoveredIntents(ctx context.Context) {
 		return
 	}
 
-	var intents []cloudclient.IntentInput
+	intentsByNamespace := make(map[string][]cloudclient.IntentInput)
```

Comment:
Say, why group them up by namespace and send each namespace individually, instead of just sending all intents at once?

Reply:
So now they are all uploaded at once